<ui version="4.0" >
 <author>Oscar Campos</author>
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>4095</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Backharddi-Monitor 1.0</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="application.qrc" >:/icons/MainWindow/pango128x128.png</iconset>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="headerBanner" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="cursor" >
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy" >
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="toolTip" >
       <string>Visit us at: http://backharddi.ideseneca.es</string>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="pixmap" >
       <pixmap resource="application.qrc" >:/icons/MainWindow/trac_banner.png</pixmap>
      </property>
      <property name="scaledContents" >
       <bool>false</bool>
      </property>
      <property name="alignment" >
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="openExternalLinks" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QWidget" native="1" name="widget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="TabWidget" name="monitorTab" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="acceptDrops" >
          <bool>true</bool>
         </property>
         <property name="autoFillBackground" >
          <bool>true</bool>
         </property>
         <property name="elideMode" >
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="usesScrollButtons" >
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="client" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <attribute name="title" >
           <string>&lt;no name></string>
          </attribute>
          <attribute name="icon" >
           <iconset resource="application.qrc" >:/icons/MainWindow/pango32x32.png</iconset>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="windowModality" >
    <enum>Qt::NonModal</enum>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4095</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadSQL" />
    <addaction name="actionCloseTask" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menu_Config" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>&amp;Config tools</string>
    </property>
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="separator" />
    <addaction name="actionEdit" />
    <addaction name="actionRemove" />
   </widget>
   <widget class="QMenu" name="menu_SQLConfig" >
    <property name="title" >
     <string>&amp;Monitor Config</string>
    </property>
    <addaction name="actionSQLConfig" />
    <addaction name="actionTaskConfig" />
   </widget>
   <widget class="QMenu" name="menu_Run" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>&amp;Run</string>
    </property>
    <addaction name="actionThrow" />
    <addaction name="actionThrow_All" />
    <addaction name="separator" />
    <addaction name="actionReboot" />
    <addaction name="actionReboot_All" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionUserManual" />
    <addaction name="separator" />
    <addaction name="actionAbout" />
    <addaction name="actionAbout_Qt4" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Config" />
   <addaction name="menu_Run" />
   <addaction name="menu_SQLConfig" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QDockWidget" name="dockClients" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="windowTitle" >
    <string>Clients</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetClients" />
  </widget>
  <widget class="QDockWidget" name="dockGroups" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="windowTitle" >
    <string>Groups</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetGroups" />
  </widget>
  <action name="actionLoad" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/load.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Load config task...</string>
   </property>
  </action>
  <action name="actionLoadSQL" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/database.png</iconset>
   </property>
   <property name="text" >
    <string>Load config task &amp;from SQL...</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/save.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save config task</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionRemove" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/editdelete.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Remove group </string>
   </property>
  </action>
  <action name="actionEdit" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/edit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Edit group config</string>
   </property>
  </action>
  <action name="actionSQLConfig" >
   <property name="text" >
    <string>&amp;SQL Configuration</string>
   </property>
  </action>
  <action name="actionTaskConfig" >
   <property name="text" >
    <string>&amp;Tasks Options</string>
   </property>
  </action>
  <action name="actionThrow" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/build.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Throw group </string>
   </property>
  </action>
  <action name="actionReboot" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/reload.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Reboot group </string>
   </property>
  </action>
  <action name="actionReboot_All" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Reboo&amp;t All</string>
   </property>
  </action>
  <action name="actionThrow_All" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/down.png</iconset>
   </property>
   <property name="text" >
    <string>Throw &amp;All</string>
   </property>
  </action>
  <action name="actionUserManual" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/easymoblog.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;User Manual</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/MainWindow/pango32x32.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;About Backharddi-Monitor</string>
   </property>
  </action>
  <action name="actionAbout_Qt4" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/MainWindow/pango32x32.png</iconset>
   </property>
   <property name="text" >
    <string>About &amp;Qt4</string>
   </property>
  </action>
  <action name="actionCloseTask" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="application.qrc" >:/icons/button_cancel.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Close config task</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global" >tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections/>
</ui>
